src/tasks/netSdk/netSdkTaskUtils.ts

```diff
@@ -33,25 +33,20 @@ export type RidCpuArchitecture =
 
 export const NetSdkRunTaskType = 'dotnet-container-sdk';
 const NetSdkDefaultImageTag = 'dev'; // intentionally default to dev tag for phase 1 of this feature
-const ErrMsgNoWorkplaceFolder = l10n.t(`Unable to determine task scope to execute task ${NetSdkRunTaskType}. Please open a workspace folder.`);
 
 export async function getNetSdkBuildCommand(context: DockerTaskExecutionContext): Promise<string> {
 
     const configuration = 'Debug'; // intentionally default to Debug configuration for phase 1 of this feature
-    const projPath = await inferProjPath(context.actionContext, context.folder);
-
     // {@link https://github.com/dotnet/sdk-container-builds/issues/141} this could change in the future
-    const publishFlag = NetCoreTaskHelper.isWebApp(projPath) ? '-p:PublishProfile=DefaultContainer' : '/t:PublishContainer';
-
-    const folderName = await quickPickWorkspaceFolder(context.actionContext, ErrMsgNoWorkplaceFolder);
+    const publishFlag = NetCoreTaskHelper.isWebApp(context.folder.uri.fsPath) ? '-p:PublishProfile=DefaultContainer' : '/t:PublishContainer';
```

Comment:
Not specifically related to this PR, but we should use our MSBuild project config targets to identify Web Apps rather than the current isWebApp method.